src/Integration/Resources/Strings.resx

```diff
@@ -359,12 +359,6 @@ ocalization note: '[try again]()' is a special markup used to translate the text
     <value>Could not acquire service: {0}</value>
     <comment>Exception message indicating the package could not acquire the given service. {0} service name</comment>
   </data>
-  <data name="ServerNuGetTrustWarningMessage" xml:space="preserve">
```

Comment:
Unrelated: the code that used this resource string has been deleted, but the resource was left behind.